_infra/helm/action/values.yaml

```diff
@@ -38,6 +43,14 @@ crons:
     name: action-plan-executor
     cron: "* * * * *"
     target: "actionplans/execute"
+  recordDeleter:
+    name: old-record-deleter
```

Comment:
Would it be an easy change to rename both the cronjobs to `action-old-record-deleter` and `action-export-scheduler`?

If not, we can do it as another bit of work down the line

Reply:
Yeah np